### ISSUE TITLE
Dependency parser README pipeline broken link

### ISSUE THREAD

**Issue by aurghob** [CogComp/cogcomp-nlp#715]
The README for the [dependency parser](https://github.com/CogComp/cogcomp-nlp/blob/master/depparse/README.md) has a broken(404) link when clicking on "cogcomp-nlp-pipeline"

**Comment by aurghob** [comment]
Meant to link to https://github.com/CogComp/cogcomp-nlp/tree/master/pipeline instead of https://github.com/CogComp/cogcomp-nlp/blob/master/depparse/pipeline/README.md?

**Comment by danyaljj** [comment]
Yup, thanks for pointing it out. PRs are welcome, if you wanna contribute!

**Comment by aurghob** [comment]
Sure! I've create a [pull request](https://github.com/CogComp/cogcomp-nlp/pull/716). 
Besides, could you please answer some questions I have on [issue 181](https://github.com/CogComp/cogcomp-nlp/issues/181)?
